torch/testing/_internal/common_methods_invocations.py

```diff
@@ -4202,6 +4205,7 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
                    dtypes=all_types_and(torch.bfloat16),
                    dtypesIfCUDA=all_types_and(torch.half, torch.bfloat16),
                    assert_autodiffed=True,
+                   supports_forward_ad=True,
```

Comment:
Are all the variants of clamp being tested? I see another clamp OpInfo above that might also need to be added?

Reply:
If `supports_forward_ad=False` for the other variants, then we test that it raises a nice error saying this is not implemented. All the tests are green, so it should be ok.